tests/runner/test_runner.py

```diff
@@ -42,6 +42,23 @@ def test_get_mode_command(self):
                 'exec_command': 'sb exec',
                 'expected_command': 'sb exec',
             },
+            {
+                'mode': {
+                    'name': 'local',
+                },
+                'exec_command': 'sb exec',
+                'expected_command': 'sb exec',
+            },
+            {
+                'mode': {
+                    'name': 'local',
+                    'proc_num': 8,
+                    'proc_rank': 6,
+                    'prefix': 'NVIDIA_VISIBLE_DEVICES={proc_rank} numactl -c $(({proc_rank}/2))'
```

Comment:
The hardcode is very limited to specific platform. Would you please design a more flexible way?

Reply:
added another case